src/util/logger.ts

```diff
@@ -29,73 +29,48 @@ export function setLogLevel(l: LogLevel): void {
 }
 
 export const logger = {
-  trivial: (message: string): void => {
+  trivial: (...messages: unknown[]): void => {
```

Comment:
👍 This is the first time I noticed that TypeScript has an `unknown` type. I think it's appropriate to place it here as the unknown allows less freedom.

https://mariusschulz.com/blog/the-unknown-type-in-typescript

Can you tell me when to use `unknown` is appropriate?

Reply:
If the type is defined and used, it is considered safer to use as unknown, so I developed as above.


This is the appropriate part of the URL you sent..!
https://mariusschulz.com/blog/the-unknown-type-in-typescript#example-reading-json-from-localstorage